src/lib/core/crash.c

```diff
@@ -382,7 +382,7 @@ crash_report_feedback_daemon(struct crash_info *cinfo)
 	snprintf_safe("}");
 	snprintf_safe("}");
 
-	pr_debug("crash dump: %s", head);
+	pr_debug("crash dump for feedback daemon: %s", head);
```

Comment:
I don't see any point in this change: debug logging is for developers - it doesn't need to be user-friendly and should be disabled in production.